### PR TITLE
Escape annotations in comments

### PR DESCRIPTION
**Pull request by cmfcmf**
| Q | A |
| --- | --- |
| Fixed tickets | --- |
| Referenced tickets | #13089 |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Kernel.php [hunk 1]
```diff
@@ -248,7 +248,7 @@ public function getBundle($name, $first = true)
      *
      * The resource name must follow the following pattern:
      *
-     *     @<BundleName>/path/to/a/file.something
+     *     "@BundleName/path/to/a/file.something"
      *
      * where BundleName is the name of the bundle
      * and the remaining part is the relative path in the bundle.
```
Comment (WouterJ) [inline src/Symfony/Component/HttpKernel/Kernel.php]: this complete doc block should actually be replaced with `{@inheritDoc}`

Comment (cmfcmf) [inline src/Symfony/Component/HttpKernel/Kernel.php]: This docblock has one RuntimeException more than the docblock in the interface. Shall I still replace it? Shall I add the additional exception to the Interface docblock?

Comment (WouterJ) [inline src/Symfony/Component/HttpKernel/Kernel.php]: /**
     * {@inheritDoc}
     *
     * @Throws RuntimeException When ...
     */

Comment (cmfcmf) [inline src/Symfony/Component/HttpKernel/Kernel.php]: Done. I've merged the two docblocks.

Comment (cmfcmf) [inline src/Symfony/Component/HttpKernel/Kernel.php]: The Fabbot says I should use `{@inheritDoc}` with a lower-cased `d`. Which one is correct now?

**Comment by fabpot** [comment]
Thank you @cmfcmf.

File: src/Symfony/Component/HttpKernel/Kernel.php (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Component/HttpKernel/Kernel.php]: @cmfcmf The lowercased one would have been correct.

**Pull request merged**